app/src/main/res/values-sv/strings.xml

```diff
@@ -0,0 +1,952 @@
+    <string name="drawer_title_sign_in">Logga in</string>
+    <string name="drawer_title_profile">Profil</string>
+    <string name="drawer_title_sign_out">Logga ut</string>
+
+    <string name="drawer_section_extras">Extra</string>
+    <string name="drawer_title_share">Dela</string>
+    <string name="drawer_title_update">Uppdatera</string>
+    <string name="layout_empty_response">Ingen data att visa</string>
+
+    <!-- Application Tips-->
+    <string name="tip_options_text">Få tillgång till programinställningarna, hjälp, changelog och postaktivitetsstatus. \n\nTryck här för att avvisa</string>
+    <string name="tip_options_title">Valmeny</string>
+    <string name="tip_search_title">Sökruta</string>
+    <string name="tip_search_message">Sök efter olika anime &amp; manga listningar, sökbar med namns nyckelord. \n\nTap här för att avvisa</string>
+    <string name="tip_navigation_title">Navigering</string>
+    <string name="tip_navigation_text">Öppna navigeringslådan för att visa annat innehåll i programmet. \n\nTryck här för att avvisa</string>
+    <string name="tip_series_options_message">Favorit, lägg till i din lista och dela den här nuvarande listan med vänner. \n\nTryck här för att avvisa</string>
+    <string name="tip_series_options_title">Alternativ tillgängligt</string>
+    <string name="tip_character_options_title">Favorit</string>
+    <string name="tip_character_options_message">Växla status för denna karaktär. \n\nTryck här för att avvisa</string>
+    <string name="tip_messages_title">Meddelanden</string>
+    <string name="tip_messages_text">Visa alla dina meddelanden som skickas till dig och de som du skickade. \n\nTryck här för att avvisa</string>
+    <string name="tip_status_post_title">Skapa statuspost</string>
+    <string name="tip_status_post_text">Lägg in ett statusmeddelande för alla att se. \n\nTryck här för att avvisa</string>
+    <string name="tip_notifications_title">Notiser</string>
+    <string name="tip_notifications_text">Få tillgång till alla meddelanden från ett ställe. \n\nTryck här för att avvisa</string>
+    <string name="tip_more_actions_title">Fler åtgärder</string>
+    <string name="tip_more_actions_text">Tryck här för fler alternativ, t.ex. visning av liknande. \ n \ nTryck här för att avvisa</string>
+    <string name="tip_compose_message_title">Skriv meddelandet</string>
+    <string name="tip_compose_message_text">Skicka ett meddelande till den nuvarande användaren. \n\ nTryck här för att avvisa</string>
+
+    <!--Media attachments on the reply activity-->
+    <string name="attach_media">Infoga media</string>
+    <string name="attach_link_title">Infoga webblänk</string>
+    <string name="attach_link_text">Ange webbadress t.ex. https://anilist.co/forum/thread/2459/1</string>
+    <string name="attach_image_title">Infoga bildlänk</string>
+    <string name="attach_image_text"Ange bildadress, t ex: https:/media.giphy.com/media/l41YxdIcDU58r5CDK/giphy.gif</string>
```

Comment:
There's a missing `>` in this line
```suggestion
    <string name="attach_image_text">Ange bildadress, t ex: https:/media.giphy.com/media/l41YxdIcDU58r5CDK/giphy.gif</string>
```

Reply:
Sorry did not see that there thanks for having sharp eyes!😊